psl-clj/pom.xml

```diff
@@ -0,0 +1,172 @@
+  <artifactId>psl-clj</artifactId>
+  <packaging>jar</packaging>
+  <name>psl-clj</name>
+  <description>A clojure user interface for PSL</description>
+  <parent>
+    <groupId>org.linqs</groupId>
+    <artifactId>psl-experimental</artifactId>
+    <version>2.1.0-SNAPSHOT</version>
+  </parent>
+  <distributionManagement>
+    <repository>
+      <id>clojars</id>
+      <name>Clojars repository</name>
+      <url>https://clojars.org/repo</url>
+    </repository>
+  </distributionManagement>
+  <build>
+    <sourceDirectory>src</sourceDirectory>
+    <testSourceDirectory>test</testSourceDirectory>
+    <resources>
+      <resource>
+        <directory>resources</directory>
+      </resource>
+    </resources>
+    <testResources>
+      <testResource>
+        <directory>resources</directory>
+      </testResource>
+    </testResources>
+    <directory>target</directory>
+    <outputDirectory>target/classes</outputDirectory>
+    <plugins>
+      <plugin>
+        <groupId>com.theoryinpractise</groupId>
+        <artifactId>clojure-maven-plugin</artifactId>
+        <version>1.8.1</version>
+        <extensions>true</extensions>
+        <configuration>
+	  <sourceDirectories>
+	    <sourceDirectory>src</sourceDirectory>
+	  </sourceDirectories>
+        </configuration>
+        <executions>
+	  <execution>
+	    <id>compile-clojure</id>
+	    <phase>compile</phase>
+	    <goals>
+	      <goal>compile</goal>
+	    </goals>
+	  </execution>
+	</executions>
+      </plugin>
+    </plugins>
+  </build>
+  <repositories>
+    <repository>
+      <id>clojars</id>
+      <url>https://clojars.org/repo/</url>
+      <snapshots>
+        <enabled>true</enabled>
+      </snapshots>
+      <releases>
+        <enabled>true</enabled>
+      </releases>
+    </repository>
+  </repositories>
+  <dependencies>
+    <dependency>
+      <groupId>org.linqs</groupId>
+      <artifactId>psl-core</artifactId>
+      <version>${psl-core-version}</version>
+    </dependency>
+    <dependency>
+      <groupId>org.linqs</groupId>
+      <artifactId>psl-groovy</artifactId>
+      <version>${psl-core-version}</version>
+    </dependency>
+    <dependency>
+      <groupId>org.clojure</groupId>
+      <artifactId>clojure</artifactId>
+      <version>1.7.0</version>
+    </dependency>
+    <dependency>
+      <groupId>org.clojure</groupId>
+      <artifactId>tools.logging</artifactId>
+      <version>0.3.1</version>
+    </dependency>
+    <dependency>
+      <groupId>org.clojars.alexmemory</groupId>
+      <artifactId>clj-util</artifactId>
+      <version>0.1.0</version>
+    </dependency>
+    <dependency>
+      <groupId>incanter</groupId>
+      <artifactId>incanter-core</artifactId>
+      <version>1.5.6</version>
+      <exclusions>
+        <exclusion>
+	  <groupId>net.sourceforge.jplasma</groupId>
+	  <artifactId>jplasma</artifactId>
+        </exclusion>
+      </exclusions>
+    </dependency>
+    <dependency>
+      <groupId>edu.emory.mathcs</groupId>
```

Comment:
Do you need all these deps?
I have a feeling that they were probably just copied over from an older version.
You probably only need a similar dependency set to the CLI/Groovy interfaces (but of course swapping Groovy for Clojure):
https://github.com/eriq-augustine/psl/blob/develop/psl-groovy/pom.xml#L33

Reply:
Done